etna/transforms/timestamp/holiday.py

```diff
@@ -83,14 +139,29 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         -------
         :
             pd.DataFrame with added holidays
+        Raises
```

Comment:
- Add one blank line before "Raises".
- You should split this in two different ValueError (like in this method https://github.com/etna-team/etna/blob/a8efed8a11b1a09c17d763f165e88c3baa689e80/etna/transforms/decomposition/deseasonal.py#L133)